group_vars/all

```diff
@@ -111,7 +115,9 @@ galaxy_config:
     dynamic_proxy_external_proxy: true
     dynamic_proxy_prefix: gie_proxy
   "uwsgi":
-    master: True
+    master: True # whether this retains sense has to be tested. It works throught
+    module: galaxy.webapps.galaxy.buildapp:uwsgi_app()
+    logfile-chmod: 644
```

Comment:
This should be more restrictive, it can contain secrets. 600 ?

Reply:
sure ! this is temporally more convenient pour debugging ;-) To be kept in mind !